src/compiler/types.ts

```diff
@@ -1693,6 +1693,8 @@ namespace ts {
           * @param path The path to test.
           */
         fileExists(path: string): boolean;
+
+        readFile(path: string): string;

```

Comment:
This breaks us in at least one place:

  https://github.com/angular/angular/blob/master/tools/@angular/tsc-wrapped/src/tsc.ts#L70

What is the timing of this change? Will it be part of 2.0? If so this will be a challenge for us.